server/index.js

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable no-unused-vars */
```

Comment:
This needs to be merged into app.js -- since that's where the backend is getting the instance of `app` from. This is probably important to do ASAP because right now server/app.js and server/index.js are both creating separate instances